data/mods/blazemod/blaze_blob_parts.json

```diff
@@ -3129,12 +1674,8 @@
     "damage_modifier": 10,
     "folded_volume": 1,
     "breaks_into": [ { "item": "sparkiegrow" } ],
-    "requirements": {
-      "repair": {
-        "time": 30000,
-        "using": [ [ "blobrep", 1 ] ]
-      }
-    },
+    "description": "A living blob turned into a vehicle weapon.",
```

Comment:
Is this missing the "heavy" part? Most other descriptions are `heavy vehicle weapon`